neo/io/plexonio.py

```diff
@@ -241,17 +241,18 @@ def read_segment(self, lazy=False, cascade=True, load_spike_waveform=True):
         for chan, h in iteritems(eventHeaders):
             if lazy:
                 times = []
+                labels = None
```

Comment:
Yes but in case of lazy load the len need to be 0 because in case of full load labels is not None but an array.
This is specifyed nowhere but in for me in case of lazyload : each array need to be of size 0 when exist at full load.

The IO support labels for EventArray so labels = [ ].
For other IOs that have EventArray.labels = None even at fullload so when lazyload EventArray.labels = None.

I am not really sur to clear, sorry.